src/test/java/pages/AuthenticationPage.java

```diff
@@ -0,0 +1,88 @@
+package pages;
+
+import baseEntities.BasePage;
+import core.ReadProperties;
+import io.qameta.allure.Step;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AuthenticationPage extends BasePage {
+
+    @FindBy(xpath = "//input[@name = 'email_create']")
+    private WebElement createAnAccountEmailAddressInput;
+    @FindBy(id = "SubmitCreate")
+    private WebElement createAnAccountButton;
+    @FindBy(xpath = "//input[@name = 'email']")
+    private WebElement alreadyRegisteredEmailAddressInput;
+    @FindBy(xpath = "//input[@name = 'passwd']")
+    private WebElement alreadyRegisteredPasswordInput;
+    @FindBy(id = "SubmitLogin")
+    private WebElement signInButton;
+    @FindBy(css = ".alert.alert-danger")
+    private WebElement errorText;
+
+    public AuthenticationPage(boolean openPageByURL) {
+        super(openPageByURL);
+    }
+
+    @Override
+    protected void openPage() {
+        driver.get("http://automationpractice.com/index.php?controller=authentication&back=my-account");
```

Comment:
in that case, it is better to store base url somewhere, and just specify an extra part in code, doing that you can easily switch hosts for testing